tap_google_ads/client.py

```diff
@@ -1,17 +1,104 @@
+import os
+import json
+import functools
+from google.auth.transport.requests import Request
+from requests import Session
 from google.ads.googleads.client import GoogleAdsClient
+from google.oauth2 import service_account
 
+def _initialize_credentials_decorator(func):
```

Comment:
Can we try to import this function from the https://github.com/googleads/google-ads-python/blob/0f0fde31d39c0b99c91112b86462d1ffd2e912cf/google/ads/googleads/oauth2.py#L30

```python
from google.ads.googleads.oauth2 import _initialize_credentials_decorator
```